ansible/roles/polkadot-validator/templates/polkadot.service.j2

```diff
@@ -30,6 +30,7 @@ ExecStart=/usr/local/bin/polkadot \
   {% endif %}
 
 Restart=always
+RestartSec=120
```

Comment:
120 is too high, i've asked Parity devs and 60s should be enough. Have you tested it?
```suggestion
RestartSec=60
```